preferencesfx/src/main/java/com/dlsc/preferencesfx/PreferencesFx.java

```diff
@@ -4,25 +4,16 @@
 import com.dlsc.preferencesfx.history.History;
 import com.dlsc.preferencesfx.model.Category;
 import com.dlsc.preferencesfx.model.PreferencesFxModel;
-import com.dlsc.preferencesfx.util.SearchHandler;
-import com.dlsc.preferencesfx.util.StorageHandler;
-import com.dlsc.preferencesfx.util.StorageHandlerImpl;
-import com.dlsc.preferencesfx.view.BreadCrumbPresenter;
-import com.dlsc.preferencesfx.view.BreadCrumbView;
-import com.dlsc.preferencesfx.view.CategoryController;
-import com.dlsc.preferencesfx.view.CategoryPresenter;
-import com.dlsc.preferencesfx.view.CategoryView;
-import com.dlsc.preferencesfx.view.NavigationPresenter;
-import com.dlsc.preferencesfx.view.NavigationView;
-import com.dlsc.preferencesfx.view.PreferencesFxDialog;
-import com.dlsc.preferencesfx.view.PreferencesFxPresenter;
-import com.dlsc.preferencesfx.view.PreferencesFxView;
-import com.dlsc.preferencesfx.view.UndoRedoBox;
+import com.dlsc.preferencesfx.util.*;
```

Comment:
we follow the Google Java Code Style guidelines, which do not allow wildcard imports (https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports)

Reply:
ok